cloudconfig/podcfg/podcfg.go

```diff
@@ -408,7 +405,9 @@ func NewBootstrapControllerPodConfig(config controller.Config, controllerName, s
 	}
 	pcfg.Jobs = []multiwatcher.MachineJob{
 		multiwatcher.JobManageModel,
-		multiwatcher.JobHostUnits,
+	}
+	if pcfg.Bootstrap.HostedModelConfig != nil && len(pcfg.Bootstrap.HostedModelConfig) > 0 {
```

Comment:
Don't need this check - the controller model can always host units

Reply:
fixed